docs/quickstart/regression.md

```diff
@@ -0,0 +1,122 @@
+---
+jupytext:
+  text_representation:
+    extension: .md
+    format_name: myst
+    format_version: 0.13
+    jupytext_version: 1.14.1
+kernelspec:
+  display_name: Python 3 (ipykernel)
+  language: python
+  name: python3
+---
+
+# Regression
+
++++
+
+Regression analysis is a set of statistical methodologies for determining the relationship between a dependent (or outcome) variable and one or more independent variables (also known as predictor variables). The most common algorithm used is Linear Regression. 
+
+Linear Regression is a supervised learning technique that attempts to fit the best line that reduces the discrepancies between the predicted and the actual target value. This best fit line is arrived at by aiming to reduce the MSE (Mean Square Error).
+
++++
+
+Let's import the required libraries and read the dataset.
+
+```{code-cell} ipython3
+import urllib.request
```

Comment:
please organize the [imports](https://peps.python.org/pep-0008/#imports):

```

    Standard library imports.
    Related third party imports.
    Local application/library specific imports.

```

Reply:
Moved imports to relevant cells.